pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -110,6 +115,7 @@ type ApplicationBackupVolumeInfo struct {
 	StorageClass             string                      `json:"storageClass"`
 	Provisioner              string                      `json:"provisioner"`
 	VolumeSnapshot           string                      `json:"volumeSnapshot"`
+	VolumeJobSecurityContext VolumeJobSecurityContext    `json:",inline"`
```

Comment:
Please add comment here to describe what is this for. These are not very obvious elements that can self explain.